### PR TITLE
Added PHP requirement to top of install doc.

### PR DESCRIPTION
**Pull request by rightsaidjames**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Installation
 
+If PHP is not installed on your environment, install it now; PHP is a requirement for Dock CLI.
```
Comment (andytson-inviqa) [inline docs/install.md]: should a version suggestion/requirement be added too?

Comment (rightsaidjames) [reply docs/install.md]: Sure, any suggestions for what the minimum should be? FWIW, PHP 5.x (Ubuntu 14.04) seemed to work fine for me, both in terms of the Docker install and the setup of an Inviqa project using the tool.

Comment (sroze) [reply docs/install.md]: As long as it's PHP 5 it should work so... I believe that we don't need it 👍

Comment (andytson-inviqa) [reply docs/install.md]: the composer.lock says >=5.3.9 as the highest constraint for PHP

Comment (andytson-inviqa) [reply docs/install.md]: and obv namespaces come in 5.3

Comment (andytson-inviqa) [reply docs/install.md]: I see short array notation in use in the source as well, so >= 5.4

Comment (sroze) [reply docs/install.md]: Good point. But as these versions are not maintained anymore, nobody should use them anyway. To summarise, I agree either way :)

Comment (andytson-inviqa) [reply docs/install.md]: They are maintained tho, see RHEL/CentOS 6/7 for example

Comment (andytson-inviqa) [reply docs/install.md]: since the code uses php syntax later than >=5.3.9, I'd suggest the composer.json say the PHP constraint to

Comment (andytson-inviqa) [reply docs/install.md]: however if you want, you can just say >= 5.6

Comment (rightsaidjames) [reply docs/install.md]: Updated 👍

File: docs/install.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Installation
 
+If PHP (>= 5.6) is not installed on your environment, install it now; PHP is a requirement for Dock CLI.
```
Comment (andytson-inviqa) [inline docs/install.md:3]: Also php pecl SSH extension is needed, which isn't a standard extension people have installed

Comment (sroze) [reply docs/install.md:3]: This will be automatically installed on OSX with `brew`.

Comment (andytson-inviqa) [reply docs/install.md:3]: nope, I don't have it installed, also Debian/Ubuntu/Fedora/Arch/CentOS etc?

Comment (rightsaidjames) [reply docs/install.md:3]: Is this relevant for someone who is just installing the tool rather than contributing to it?

Comment (andytson-inviqa) [reply docs/install.md:3]: SSH is for someone using the tool, not testing

Comment (andytson-inviqa) [reply docs/install.md:3]: ah, by "automatically installed", @sroze means dock-cli will install it

Comment (andytson-inviqa) [reply docs/install.md:3]: so no need to mention it I guess

**Comment by sroze** [comment]
Danke!

**Pull request merged**